Marlin/language_fr.h

```diff
@@ -189,7 +192,6 @@
 #if LCD_WIDTH > 19
   #define MSG_INFO_PRINT_COUNT              "Nbre impressions"
   #define MSG_INFO_COMPLETED_PRINTS         "Terminees"
-  #define MSG_INFO_PRINT_TIME               "Tps impr. total"
```

Comment:
i'm using git from command line under debian jessie

what I don't understand is that the mentionned line is not deleted on my local file, neither on my repo, but it's still noted as deleted in this PR

ps : note that, for this particular string, it won't be very anoying, because english and french translation are the same